### ISSUE TITLE
why I got the negative loss ?

### ISSUE THREAD

**Issue by zzdang** [jakeret/tf_unet#149]
I got the nagetive loss use the unet ,I don't know what's  the wrong in my code.
I use the gt (maks) of 2 class,background and foregroud,with the mask pixels are 255 .The mask png is 1 channel.
I use image_util.ImageDataProvider to provide data,unet.Unet is the network.
Thank you for helping me

**Comment by PhySci** [comment]
Do you use "class_weights" in the model definition?

**Comment by jakeret** [comment]
Depending on the loss you are using this actually [valid](https://github.com/jakeret/tf_unet/blob/master/tf_unet/unet.py#L235)

**Comment by PhySci** [comment]
I see that cross-entropy loss function gives negative value. Any ideas?

**Comment by jakeret** [comment]
I see, that's every odd. Haven't seen that before
